onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -396,6 +398,8 @@ def _where_input_wrangler(
     skip("empty_like", reason="Using zeros_like to simulate empty_like"),
     xfail("logcumsumexp", reason="naive implementation not numerically stable"),
     xfail("logsumexp", reason="ONNX Runtime 1.13 does not support ReduceLogSumExp-18"),
+    xfail("new_empty", reason="Using zeros to simulate empty"),
```

Comment:
@xiaowuhu I also realized this can succeed unexpectedly. In that case a skip may be a better choice than xfail.